documentation/src/sphinx/WSQuickStart.rst

```diff
@@ -126,5 +126,4 @@ works with:
 
     WS.url("https://example.com").get()
 
-You can see more examples on the [[example
-configurations\|ExampleSSLConfig]] page.
+You can see more examples on the :ref:`example configurations\ <ExampleSSLConfig>` page.
```

Comment:
The previously here `|` was a bit weird, so got curious.
Thanks for checking :)